client/src/api/planters.js

```diff
@@ -6,14 +6,36 @@ import {
 import {session} from "../models/auth";
 
 export default {
-  getPlanter(id){
-    const query = `${process.env.REACT_APP_API_ROOT}/api/${getOrganization()}planter/${id}`;
-    return fetch(query, {
-      method: "GET",
-      headers: { 
-        "content-type": "application/json" ,
-        Authorization: session.token ,
-      },
+  getPlanter(id) {
+    const apiRoot = `${process.env.REACT_APP_API_ROOT}/api`;
+    const planterQuery = `${apiRoot}/${getOrganization()}planter/${id}`;
+    const registrationQuery = `${apiRoot}/planter_registration/?filter=where[planterId]=${id}`;
+
+    return Promise.all([
+      fetch(planterQuery, {
+        method: "GET",
+        headers: { 
+          "content-type": "application/json" ,
+          Authorization: session.token ,
+        },
+      }),
+      fetch(registrationQuery, {
+        method: "GET",
+        headers: { 
+          "content-type": "application/json" ,
+          Authorization: session.token ,
+        },
+      })
+    ])
+    .then(responses => {
+      let planter = responses[0]
```

Comment:
I think this can be `const` right? you're just mutating it not reassigning

Reply:
Yes, but it's been removed altogether in a subsequent commit.